src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -121,26 +121,134 @@ public net.minecraft.client.particle.ParticleEngine$SpriteParticleRegistration
+public net.minecraft.client.renderer.RenderStateShard COLOR_DEPTH_WRITE
+public net.minecraft.client.renderer.RenderStateShard COLOR_WRITE
+public net.minecraft.client.renderer.RenderStateShard DEPTH_WRITE
+public net.minecraft.client.renderer.RenderStateShard NO_LAYERING
+public net.minecraft.client.renderer.RenderStateShard POLYGON_OFFSET_LAYERING
+public net.minecraft.client.renderer.RenderStateShard VIEW_OFFSET_Z_LAYERING
+public net.minecraft.client.renderer.RenderStateShard MAIN_TARGET
+public net.minecraft.client.renderer.RenderStateShard OUTLINE_TARGET
+public net.minecraft.client.renderer.RenderStateShard TRANSLUCENT_TARGET
+public net.minecraft.client.renderer.RenderStateShard PARTICLES_TARGET
+public net.minecraft.client.renderer.RenderStateShard WEATHER_TARGET
+public net.minecraft.client.renderer.RenderStateShard CLOUDS_TARGET
+public net.minecraft.client.renderer.RenderStateShard ITEM_ENTITY_TARGET
+public net.minecraft.client.renderer.RenderStateShard DEFAULT_LINE
+public net.minecraft.client.renderer.RenderStateShard NO_COLOR_LOGIC
+public net.minecraft.client.renderer.RenderStateShard OR_REVERSE_COLOR_LOGIC
 protected-f net.minecraft.client.renderer.RenderStateShard setupState # setupState
+protected-f net.minecraft.client.renderer.RenderStateShard$TextureStateShard blur # blur
+protected-f net.minecraft.client.renderer.RenderStateShard$TextureStateShard mipmap # mipmap
+public net.minecraft.client.renderer.RenderStateShard setupGlintTexturing(F)V
 public net.minecraft.client.renderer.RenderStateShard$BooleanStateShard
+public net.minecraft.client.renderer.RenderStateShard$ColorLogicStateShard
 public net.minecraft.client.renderer.RenderStateShard$CullStateShard
 public net.minecraft.client.renderer.RenderStateShard$DepthTestStateShard
 public net.minecraft.client.renderer.RenderStateShard$EmptyTextureStateShard
 public net.minecraft.client.renderer.RenderStateShard$LayeringStateShard
 public net.minecraft.client.renderer.RenderStateShard$LightmapStateShard
+public net.minecraft.client.renderer.RenderStateShard$LineStateShard
 public net.minecraft.client.renderer.RenderStateShard$MultiTextureStateShard
 public net.minecraft.client.renderer.RenderStateShard$OffsetTexturingStateShard
 public net.minecraft.client.renderer.RenderStateShard$OutputStateShard
 public net.minecraft.client.renderer.RenderStateShard$OverlayStateShard
 public net.minecraft.client.renderer.RenderStateShard$ShaderStateShard
 public net.minecraft.client.renderer.RenderStateShard$TextureStateShard
-protected-f net.minecraft.client.renderer.RenderStateShard$TextureStateShard blur # blur
-protected-f net.minecraft.client.renderer.RenderStateShard$TextureStateShard mipmap # mipmap
 public net.minecraft.client.renderer.RenderStateShard$TexturingStateShard
 public net.minecraft.client.renderer.RenderStateShard$TransparencyStateShard
 public net.minecraft.client.renderer.RenderStateShard$WriteMaskStateShard
```

Comment:
You do know that AT supports * operations, for fields and methods? Specifically for this reason.

Reply:
Weird wildcards worked just fine here, I remember them not working for me in the past which is why I didn't use them here initially